src/elements/db/ElementRelationParamParser.php

```diff
@@ -90,7 +92,9 @@ public static function normalizeRelatedToParam(mixed $relatedToParam): array
             $relatedToParam = [$relatedToParam];
         }
 
-        $relatedToParam = array_map([static::class, 'normalizeRelatedToCriteria'], $relatedToParam);
+        $relatedToParam = Collection::make($relatedToParam)->map(function($relatedToParam) use ($siteId) {
+            return static::normalizeRelatedToCriteria($relatedToParam, $siteId);
+        })->toArray();
```

Comment:
@brandonkelly since we're going into `4.3` I went ahead and used a collection here, but happy to change it back to `array_map` if that's better for a reason I'm not thinking of.